packages/blaze/view.js

```diff
@@ -188,6 +188,40 @@ Blaze.View.prototype.autorun = function (f, _inViewScope) {
   return c;
 };
 
+Blaze.View.prototype._errorIfShouldntCallSubscribe = function () {
+  var self = this;
+
+  if (! self.isCreated) {
+    throw new Error("View#subscribe must be called from the created callback at the earliest");
+  }
+  if (self._isInRender) {
+    throw new Error("Can't call View#subscribe from inside render(); try calling it from the created or rendered callback");
+  }
+  if (self.isDestroyed) {
+    throw new Error("Can't call View#subscribe from inside the destroyed callback, try calling it inside created or rendered.");
+  }
+};
+
+/**
+ * Just like Blaze.View#autorun, but with Meteor.subscribe instead of
+ * Tracker.autorun. Stop the subscription when the view is destroyed.
+ * @return {SubscriptionHandle} A handle to the subscription so that you can
+ * see if it is ready, or stop it manually
+ */
+Blaze.View.prototype.subscribe = function (/* arguments */) {
+  var self = this;
+
+  self._errorIfShouldntCallSubscribe();
+
+  var subHandle = Meteor.subscribe.apply(Meteor, arguments);
```

Comment:
would be nice, in theory, to have a way to use a different connection than the default connection.  Not sure how you would work it into this varargs API.  I guess you could just have a different function `subscribeOnConnection`